docs/tutorial/managing-ontology-project.md

```diff
@@ -0,0 +1,54 @@
+1. *Principal Investigators*: Manage the projects that fund ontology curation activities.
+
+
+For an effective management of an ontology, the following criteria are recommended:
+
+1. There should be *at least one main Principal Ontology Curator* for every ontology project. The importance is not whether this curator has a specific number of hours per week allocated to the project (although based on our experience, 1 day per week is minimum), but whether the curator has a sense of ownership, i.e. they understand that they are the primary responsible person for maintaining the ontology moving forward. Because of potential grant overlapping issues, we recommend to have *at least 1 Principal Ontology Curator* for every grant/funded project that has a stake in the ontology.
+2. Every effective ontology needs at least a few hours per week from an Ontology Pipeline Developer (OPD). More on that role later. The OPD does not always have as strong a sense of ownership of the ontology project, but typically has a strong sense of responsibility to members of the curation team.
+3. There should be separate meetings for curation and technical activities - both problems are hard, and need different team members being present. We recommend at least monthly technical and biweekly curation calls, but for many of the most effective ontology projects we manage, weekly technical and weekly curation calls are normal.
+
+Without the above minimum criteria, the following recommendations will be very hard to implement.
+
+## What do we need? The Project Management Toolbox
+
+Every ontology or group of related ontologies (sometimes it is easier to manage multiple ontologies at once, because their scope or technical workflows are quite uniform or they are heavily interrelated) should have:
+
+1. *at least two teams*, Curation Team and Technical Team, with clearly defined members. We recommend to create two teams on GitHub and keep its members always up to date (i.e. remove members that are not actively participating). Many of our projects furthermore have a third "core team", which is a more liberal team containing everyone from stakeholders, principal investigators, curators and users.
+2. *two distinct project boards*: One for the Curation Team, and one for the Technical Team. The details on how to design the boards is up to the respective teams, but we found a simple 4 stage board with sections for `To Do` (issues that are important but not urgent), `Priority` (issues that are important and urgent), `In Progress` (issues that are being worked on), `Under review` (issues that need review) and `Done` (finished issues) the most useful. The least useful column on a project board is `Backlog`, as it clutters the board and creates a massive overhead in the rare times the team meets to discuss the next steps.
```

Comment:
Any advice on permissions? I'd be inclined to recommend all projects have a large general team all with write permissions to avoid overhead in managing and encourage edits/PRs.  I think fine as long as Master is protected - requiring reviews for merge. Strict management of members should only be for core teams - maybe say something about whether these teams are important for technical/permissions purposes or whether this is mostly just a social/organizational/responsibility marker.

Reply:
I agree with that. Let me clarify this a bit.